src/styles/summernote-common.scss

```diff
@@ -607,3 +607,11 @@ $img-margin-right: 10px;
     }
   }
 }
+
+/* Handle
+ ------------------------------------------ */
+html, body {
+  .summernote-fullscreen {
```

Comment:
For continuity, it would be better to name this `.note-fullscreen` to keep with our current naming convention.

Reply:
Actually, I just realised when looking at the class file that there is already .note-fullscreen, it may be better to use something like .note-fullscreen-body, essentially we shortened the summernote name (as you probably guessed).

Reply:
In editor it is ok to use shortened class names, but outside I would recommend more explicit naming. In this case anyone who see this class name on html/body elements will be able to quickly recognize where it come from.

Please, confirm that you really want this and I will update this PR.

IMHO. I would recommend using similar approach to the top level class names. `note` is too common word and there is some probability for class names conflicts.

Reply:
The full name `summernote-` is only used for the filenames, if you have a look at the class names, they are mostly commonly started with `.note-`, in the Skunkworks repository they are all updated to `.note-`, to avoid conflicts with other page styles.
I get what your saying, though, but it's not the naming convention used throughout the stylesheets, don't shoot me, I didn't come up with that originally, lol.
Also, with some testing, we don't need a lot of the script you've added, only toggling the class is needed, as I've discovered. Test in Chrome, do other editors require all the changes. I'll paste what I have working here so you can see what I mean. I did discover which made me look closer at your implementation, that the classes were not being applied to the `<html/>` or `<body/>` elements. 
````javscript
import $ from 'jquery';

export default class Fullscreen {
  constructor(context) {
    this.context = context;

    this.$editor = context.layoutInfo.editor;
    this.$toolbar = context.layoutInfo.toolbar;
    this.$editable = context.layoutInfo.editable;
    this.$codable = context.layoutInfo.codable;

    this.$window = $(window);
    this.$scrollbar = $('html, body');

    this.onResize = () => {
      this.resizeTo({
        h: this.$window.height() - this.$toolbar.outerHeight(),
      });
    };
  }

  resizeTo(size) {
    this.$editable.css('height', size.h);
    this.$codable.css('height', size.h);
    if (this.$codable.data('cmeditor')) {
      this.$codable.data('cmeditor').setsize(null, size.h);
    }
  }

  /**
   * toggle fullscreen
   */
  toggle() {
    this.$editor.toggleClass('note-fullscreen');
    this.$scrollbar.toggleClass('note-fullscreen-body'); // <--- this is all I needed to do the same functionality as you propose.
    if (this.isFullscreen()) {
      this.$editable.data('orgHeight', this.$editable.css('height'));
      this.$editable.data('orgMaxHeight', this.$editable.css('maxHeight'));
      this.$editable.css('maxHeight', '');
      this.$window.on('resize', this.onResize).trigger('resize');
    } else {
      this.$window.off('resize', this.onResize);
      this.resizeTo({ h: this.$editable.data('orgHeight') });
      this.$editable.css('maxHeight', this.$editable.css('orgMaxHeight'));
    }

    this.context.invoke('toolbar.updateFullscreen', this.isFullscreen());
  }

  isFullscreen() {
    return this.$editor.hasClass('fullscreen');
  }
}
````

Reply:
1. Class name is renamed.
2. Yes, your code does the similar thing. But my code is more bulletproof. If someone mistakenly removes this class name from html/body then your code will add it back but my code will do exactly what is expected. Also, calling `this.isFullscreen()` multiple times in a row doesn't have any sense.

Reply:
Have you checked this without your button click that toggles full screen then destroy's the editor, watching the inspector in Chrome the class is not added to the `<html/>` or `<body/>` tags?

Reply:
Please, note that the listener is added to all buttons:
```
$('button').click(() => {
        $('#summernote').summernote('fullscreen.toggle');
        $('#summernote').summernote('destroy');
    });
```